src/OrbitGl/TrackManager.h

```diff
@@ -77,8 +77,8 @@ class TrackManager {
   std::unordered_map<int32_t, std::shared_ptr<ThreadTrack>> thread_tracks_;
   std::map<std::string, std::shared_ptr<AsyncTrack>> async_tracks_;
   std::map<std::string, std::shared_ptr<GraphTrack>> graph_tracks_;
-  // Mapping from timeline hash to GPU tracks.
-  std::unordered_map<uint64_t, std::shared_ptr<GpuTrack>> gpu_tracks_;
+  // Mapping from timeline to GPU tracks. Timeline name is used for stable ordering.
```

Comment:
I would add a bit more information here. Why do we want stable ordering for example? Just from the current comment alone it's still hard to understand why I can't change this from timeline name to timeline hash. The ordering is just as stable in that case (if I use std::map), the exact order of tracks is just not what we want then. We explicitly want the ordering on timeline strings (so that tracks like "gfx" and "gfx_markers" appear next to each other in the UI).

Reply:
I extended the comment.